src/main/java/seedu/address/model/person/InterviewDate.java

```diff
@@ -22,7 +24,22 @@ public InterviewDate(String interviewDate) {
         if (interviewDate == null) {
             value = null;
         } else {
-            value = LocalDateTime.parse(interviewDate, formatter);
+            value = LocalDateTime.parse(interviewDate, DATE_FORMAT);
+        }
+    }
+    /**
+     * Checks if the entered interview date follows the format
+     */
+    public static boolean isValidDate(String test) {
+        try {
+            // Check the format of the interview date by parse
+            LocalDateTime inputDate = LocalDateTime.parse(test, DATE_FORMAT);
+            LocalDateTime currentDate = LocalDateTime.now();
+            // Set the time part of currentDate to 00:00
+            currentDate = currentDate.withHour(0).withMinute(0).withSecond(0);
+            return inputDate.isAfter(currentDate);
+        } catch (DateTimeParseException e) {
+            return false;
         }
     }
```

Comment:
The method is appropriately named isValidDate, which clearly indicates its purpose of checking if the entered interview date follows a specific format. The code is concise and easy to understand, which enhances readability and maintainability. Using a try-catch block to catch DateTimeParseException is a good practice. It handles the case where the input string does not match the expected date format.